query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/order_and_pagination/order_by_dependent_pagination.rs

```diff
@@ -235,11 +228,9 @@ mod order_by_dependent_pag {
     }
 
     // "[Circular with differing records] Ordering by related record field ascending" should "work"
-    // TODO(dom): Not working on mongo
-    // TODO(dom): Query result {"data":{"findManyModelA":[{"id":1,"b":{"c":{"a":{"id":3}}}},{"id":2,"b":{"c":{"a":{"id":4}}}}]}}
-    // TODO(dom): is not part of the expected results: ["{\"data\":{\"findManyModelA\":[{\"id\":3,\"b\":null},{\"id\":4,\"b\":null},{\"id\":1,\"b\":{\"c\":{\"a\":{\"id\":3}}}},{\"id\":2,\"b\":{\"c\":{\"a\":{\"id\":4}}}}]}}", "{\"data\":{\"findManyModelA\":[{\"id\":1,\"b\":{\"c\":{\"a\":{\"id\":3}}}},{\"id\":2,\"b\":{\"c\":{\"a\":{\"id\":4}}}},{\"id\":3,\"b\":null},{\"id\":4,\"b\":null}]}}"]
+    // TODO: Does not work on MySQL & SQLite. Figure out why?
     // TODO(julius): should enable for SQL Server when partial indices are in the PSL
-    #[connector_test(exclude(SqlServer, MySql, MongoDb))]
+    #[connector_test(exclude(SqlServer, MySql, Sqlite))]
```

Comment:
Did we talk about this? It seems strange to me that this is passing on current master.